html.go

```diff
@@ -901,31 +902,10 @@ func doubleSpace(out *bytes.Buffer) {
 }
 
 func isRelativeLink(link []byte) (yes bool) {
-	// a tag begin with '#'
-	if link[0] == '#' {
-		return true
+	u, err := url.Parse(string(link[:]))
```

Comment:
What is the purpose of creating a new slice from `link` that is equivalent to `link` itself? I.e., why do `link[:]` instead of just `link`?